app/mails/management/commands/sendmail.py

```diff
@@ -1,6 +1,4 @@
-import email
-import smtplib
-import logging
+import email, smtplib, logging, traceback, sys
```

Comment:
Style: Don't import multiple modules at once. Also, `traceback` and `sys` are not used anywhre